docs/integrations/google_colab.rst

```diff
@@ -3,38 +3,108 @@
+Make sure that you have an account with both `Google <https://support.google.com/accounts/answer/27441?hl=en>`_ and `Neptune <https://neptune.ai/register>`_.
 
-    Go to the Neptune web app and `get your API token <../python-api/how-to/organize.html#find-my-neptune-api-token>`_. Set it to the environment variable `NEPTUNE_API_TOKEN`:
+Quickstart
+----------
 
-    .. code-block:: Bash
+|Run on Colab|
 
-        env NEPTUNE_API_TOKEN='your_private_neptune_api_token=='
+Step 1: Install Neptune client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Go to your first cell in Google Colab and install ``neptune-client``:
 
-3. Delete this cell.
+.. code-block:: Bash
 
-    .. warning::
+   pip install neptune-client
 
-        It is very important that you delete this cell, so as not to share your private token with anyone.
+Step 2: Set Neptune API token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-4. Now, run your training script with Neptune:
+#. Go to the Neptune web app and `get your API token <../python-api/how-to/organize.html#find-my-neptune-api-token>`_
 
-    .. code-block:: Python
+   .. image:: https://neptune.ai/wp-content/uploads/get_token.gif
+      :target: https://neptune.ai/wp-content/uploads/get_token.gif
+      :alt: Getting your Neptune API token
+   
+#. Run the code below:
+    
+   .. code-block:: Python
 
-        import neptune
-        neptune.init('USER_NAME/PROJECT_NAME')
+      from getpass import getpass
+      api_token = getpass('Enter your private Neptune API token')
 
-        with neptune.create_experiment():
-            neptune.send_metric('auc', 0.92)
\ No newline at end of file
+#. Enter the token in the input box. This will save your token to ``api_token``
+
+Step 3: Initialize your Neptune Project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below:
+
+.. code-block:: Python
+
+    import neptune
+
+    neptune.init(project_qualified_name = 'your_user_name/your_project_name'
+                 , api_token = api_token)
+
+Step 4: Run your training script with Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below:
+
+.. code-block:: Python
+
+    from numpy import random
+    from time import sleep
+
+    with neptune.create_experiment():
+        
+        neptune.log_metric('single_metric', 0.62)
+        
+        for i in range(100):
+            sleep(0.2) # to see logging live
+            neptune.log_metric('random_training_metric', i*random.random())
+            neptune.log_metric('other_random_training_metric', 0.5*i*random.random())
+
+Step 5: Check metrics on the Neptune UI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Follow the link shown to view your experiment progress and metrics on the Neptune UI.
+
+|Explore experiment|
+
+What's next
+-----------
+
+Now that you know how to integrate Neptune with Google Colab, you can check:
+
+* `What can you log to experiments? <https://docs.neptune.ai/logging-and-managing-experiment-results/logging-experiment-data/what-can-you-log-to-experiments.html>`_
```

Comment:
As those are internal links they should be added as explained here:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

Basically, you add a link connector in a place you'd like the link to go

`.. _LINK_CONNECTOR_NAME:`

and then in a place where you want to link you do

```
:ref:`TEXT ANCHOR <LINK_CONNECTOR_NAME>`

Reply:
done